microcosm_flask/tests/conventions/test_crud.py

```diff
@@ -101,6 +101,29 @@ def test_create(self):
             },
         })
 
+    def test_create_empty_object(self):
+        response = self.client.post("/api/person", data='{}')
+        self.assert_response(response, 422, {
+            "code": 422,
+            "message": "Validation error",
+            "retryable": False,
+            "context": {
+                "errors": [{
```

Comment:
Maybe use {{contains_inanyorder}} on these errors so that ordering changes doesn't break us?